code/modules/admin/player_panel.dm

```diff
@@ -414,6 +414,7 @@
 				if(M)
 					dat += "<tr><td><a href='?_src_=holder;adminplayeropts=\ref[M]'>[M.real_name]</a>[M.client ? "" : " <i>(ghost)</i>"][M.stat == 2 ? " <b><font color=red>(DEAD)</font></b>" : ""]</td>"
 					dat += "<td><A href='?priv_msg=[M.ckey]'>PM</A></td></tr>"
+					dat += "<td><A href='?_src_=holder;adminplayerobservefollow=\ref[M]'>FLW</a></td></tr>"

```

Comment:
Pls to be using the macro
https://github.com/tgstation/tgstation/pull/19070/files#diff-a51fd60b9f94372d01da166ee2cb3401R41